contracts/interfaces/IKnowForwarderAddress.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier:MIT
+pragma solidity ^0.6.2;
+
+interface IKnowForwarderAddress {
```

Comment:
Do you think its such an esoteric case that a recipient defines its own forwarder? I think it is part of the recipient, maybe rename it to getDefaultForwarder()

Reply:
Discussed over email. Cannot use all existing channels for the same discussion.